### PR TITLE
tinymce link popup translations an upload tab

### LINKED ISSUE #49: Cannot remove upload tab from link popup
Setting upload false into data-pat-tinymce does not remove the upload button

### PR DESCRIPTION
**Pull request by parruc**
Should fix #49 #50 and #51

### CODE REVIEW

**Comment by parruc** [comment]
@jenkins-plone-org please run jobs

**Comment by parruc** [comment]
@jenkins-plone-org please run jobs

File: buildout.cfg [hunk 1]
```diff
@@ -17,10 +17,6 @@ develop = .
 
 
 
-[sources]
-Products.CMFPlone = git ${remotes:plone}/Products.CMFPlone.git pushurl=${remotes:plone_push}/Products.CMFPlone.git branch=thet-staticresources
```
[the remark this answers is not in the document]
Comment (parruc) [reply buildout.cfg]: It has been done here: https://github.com/plone/mockup/pull/928

[the remark this answers is not in the document]
Comment (thet) [reply buildout.cfg]: tnx! i'm closing this PR then.

**Pull request closed without merging**